### PR TITLE
fixing :p (take two)

### PR DESCRIPTION
**Pull request by gurfan**
should fix :p not working for ai private and updates ai private intercoms with the correct frequency
[bugfix]

:cl:
 * bugfix: The :p hotkey for the AI Private Channel should work again.

### CODE REVIEW

File: code/game/machinery/telecomms/presets.dm [hunk 1]
```diff
@@ -106,11 +106,11 @@
 /obj/machinery/telecomms/bus/preset_four
 	id = "Bus 4"
 	network = "tcommsat"
-	autolinkers = list("processor4", "engineering", "common")
+	autolinkers = list("processor4", "engineering", "common", "ai private")
```
Comment (DamianX) [inline code/game/machinery/telecomms/presets.dm]: do you know what this does?

Comment (gurfan) [reply code/game/machinery/telecomms/presets.dm]: i forgot to remove this when i realized i didnt need to make a new server, it links the tcomms machines together like an ID tag

**Comment by toomykins** [comment]
thanks for this i'm never touching maps again

**Comment by jknpj** [comment]
Doesn't adding AI Priv to tcomms defeats the purpose of being an AI Private channel?

**Comment by toomykins** [comment]
if you made it intercom-only then the :p command would have to be removed, it's just more convenient this way

**Comment by SonixApache** [comment]
:p

**Comment by DamianX** [comment]
What broke this, anyway? In other words, how did it work not passing through telecomms?

**Comment by toomykins** [comment]
> What broke this, anyway? In other words, how did it work not passing through telecomms?

when frequencies became randomized this was forgotten, mostly because the old frequency was never stated outright in the code, but rather included in a for loop of included frequencies on the machines. the for loop still exists, it is rather useless and the frequencies unnamed, which is why the old AI private frequency now seems like it's going to [common], but it's not, it's just named common.

File: code/game/machinery/telecomms/presets.dm [hunk 2]
```diff
@@ -66,7 +66,7 @@
 
 /obj/machinery/telecomms/receiver/preset_right/initialize()
 	..()
-	freq_listening = list(RESPONSE_FREQ, COMM_FREQ, ENG_FREQ, SEC_FREQ, COMMON_FREQ) //ert, command, engineering, security
+	freq_listening = list(RESPONSE_FREQ, COMM_FREQ, ENG_FREQ, SEC_FREQ, COMMON_FREQ, AIPRIV_FREQ) //ert, command, engineering, security
 	for(var/i = 1441, i < 1489, i += 2)
```
Comment (toomykins) [inline code/game/machinery/telecomms/presets.dm:70]: very useless misleading code now, with frequency randomization

**Pull request merged**